src/qr_code_generator.rs

```diff
@@ -142,12 +141,12 @@ fn inner_generate_secure_join_qr_code(
         // Text
         const BIG_TEXT_CHARS_PER_LINE: usize = 32;
         const SMALL_TEXT_CHARS_PER_LINE: usize = 38;
-        let chars_per_line = if qrcode_description.len() > SMALL_TEXT_CHARS_PER_LINE * 2 {
+        let chars_per_line = if raw_qrcode_description.len() > SMALL_TEXT_CHARS_PER_LINE * 2 {
```

Comment:
Maybe rename function argument to just `qrcode_description`?

Reply:
makes sense, i'll change that.